### PR TITLE
Fix linting issues in utils

### PR DESCRIPTION
**Pull request by a3957273**
Mostly Typescript issues, a few changes of functions -> React elements.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -25,6 +25,8 @@ module.exports = {
     'prettier',
   ],
   rules: {
+    'import/no-extraneous-dependencies': 'off',
+    'no-restricted-syntax': 'off',
```
Comment (a3957273) [inline .eslintrc.js:29]: Sometimes `for await` loops are useful.

File: .eslintrc.js [hunk 2]
```diff
@@ -25,6 +25,8 @@ module.exports = {
     'prettier',
   ],
   rules: {
+    'import/no-extraneous-dependencies': 'off',
```
Comment (a3957273) [inline .eslintrc.js:28]: This breaks our test harness linting.

File: src/Form/ModelEditSubmission.tsx [hunk 3]
```diff
@@ -20,7 +20,7 @@ const ModelEditSubmission = ({
   onSubmit: any
   activeStep: number
   setActiveStep: Function
-  modelUploading: boolean
+  modelUploading: boolean | undefined
```
Comment (ARADDCC002) [inline src/Form/ModelEditSubmission.tsx:23]: How are we handling an undefined modelUploading value here? Hard to see in this big PR!

File: utils/complianceFlow.ts [hunk 4]
```diff
@@ -4,7 +4,7 @@ import { Deployment, Version } from 'types/interfaces'
 
 function approvalColour(state: string) {
   if (state === 'No Response') return yellow['200']
-  if (state === 'Accepted') return green['A200']
+  if (state === 'Accepted') return green.A200
   if (state === 'Declined') return red['200']
```
Comment (ARADDCC002) [inline utils/complianceFlow.ts:8]: Is there a reason we're now using a different format for green and red values here?

File: utils/complianceFlow.ts [hunk 5]
```diff
@@ -13,7 +13,7 @@ function approvalColour(state: string) {
 export default function createComplianceFlow(version: Version) {
   const position: XYPosition = { x: 0, y: 0 }
 
-  const success = green['A200']
+  const success = green.A200
```
Comment (ARADDCC002) [inline utils/complianceFlow.ts:16]: Same as above

File: utils/fetcher.ts [hunk 6]
```diff
@@ -20,7 +20,9 @@ export const getErrorMessage = async (res: Response) => {
   let messageError = res.statusText
   try {
     messageError = `${res.statusText}: ${(await res.json()).message}`
-  } catch (e) {}
+  } catch (e) {
+    // unable to identify error message, possibly a network failure
```
Comment (ARADDCC002) [inline utils/fetcher.ts:24]: Should we not log or return something here instead of just a comment

Comment (a3957273) [reply utils/fetcher.ts:24]: Nah, it will then use the default message `res.statusText` which is fine for handling that error.

**Review by ARADDCC002 — APPROVED** [review]

**Pull request merged**